flycheck.el

```diff
@@ -8630,6 +8631,16 @@ See URL `http://slim-lang.com'."
           line-end))
   :modes slim-mode)
 
+(flycheck-define-checker slim-lint
+  "A SLIM linter
+
+See URL `https://github.com/sds/slim-lint'."
+  :command ("slim-lint" "--reporter=checkstyle"
+            source-original)
```

Comment:
I have no idea, I just found an example for defining custom checkers online and modified it. Probably should have gone to the source rather than random online examples though ;). It seems like source works, so that the display isn't lagging behind until you save. Will change!